content/doc/applications/docker/_index.md

```diff
@@ -204,3 +204,18 @@ We provide a few examples of dockerized applications on Clever Cloud.
 {{% content/more-config %}}
 
 {{% content/url_healthcheck %}}
+
+### Commonly encountered issues
+
+{{% details title="I can't reach my app port" closed="true" %}}
+
+You probably want to use a specific port for their apps. However, you can't directly reach for that port, as your request will go through Clever Cloud's reverse proxies, then through the VM.
+
+Here are the steps:
+* Create a TCP redirection that will allow entry through our load balancers, that will come out as 4040 or 8080 on the Docker. You can find a guide here:
+<https://developers.clever-cloud.com/doc/administrate/tcp-redirections/>
+If you are not using 4040/8080 as your app port, you must redirect those to the desired port of your app.  
+* Set environnement variable `CC_DOCKER_EXPOSED_TCP_PORT` or `CC_DOCKER_EXPOSED_HTTP_PORT` (depending on technology) to translate 4040/8080 to anyport your wish on your app.
+You can connect to your app on your custom port by connecting to it from the outside on the TCP port that has been provided. 
```

Comment:
Unclear sentence, needs to be more specific : what is connecting from where ? Maybe rephrase it like this for now:

 ```suggestion
You can connect to your app on your custom port on the provided TCP port. 
```